
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://biolink.github.io/information-resource-registry/registry/">
      
      
        <link rel="prev" href="../appendices/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.5.39">
    
    
      
        <title>Registry - Information Resource Registry Model Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.8c3ca2c6.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../style.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-J23SCD6JB8"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-J23SCD6JB8",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-J23SCD6JB8",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="black" data-md-color-primary="black" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#information-resource-registry-overview" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Information Resource Registry Model Documentation" class="md-header__button md-logo" aria-label="Information Resource Registry Model Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Information Resource Registry Model Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Registry
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/biolink/information-resource-registry" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.6.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Information Resource Registry Model Documentation" class="md-nav__button md-logo" aria-label="Information Resource Registry Model Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    Information Resource Registry Model Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/biolink/information-resource-registry" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.6.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Overview
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../data_flow_diagram.png" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Visualize Translator Data Flow (sankey)
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://biolink.github.io/information-resource-registry/infores_catalog.yaml" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    View the Infores Catalog
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../infores_catalog.yaml" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Download the Infores Catalog
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../implementation-guidance/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Implementation Guidance
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../biolink-infores-parameters/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Catalog Metamodel Details
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../appendices/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Addition information
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Registry
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Registry
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#information-resource-registry-overview" class="md-nav__link">
    <span class="md-ellipsis">
      Information Resource Registry Overview
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#information-resource-registry-overview" class="md-nav__link">
    <span class="md-ellipsis">
      Information Resource Registry Overview
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Registry</h1>

<h2 id="information-resource-registry-overview">Information Resource Registry Overview</h2>
<table>
<thead>
<tr>
<th>ID</th>
<th>Name</th>
<th>Status</th>
<th>Knowledge Level</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../resources/rnacentral/">infores:rnacentral</a></td>
<td>RNAcentral</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A comprehensive database of non-coding RNA sequenc...</td>
</tr>
<tr>
<td><a href="../resources/dictybase/">infores:dictybase</a></td>
<td>dictyBase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The central resource for Dictyostelid genomics, pr...</td>
</tr>
<tr>
<td><a href="../resources/cacao/">infores:cacao</a></td>
<td>Community Assessment of Community Annotation with Ontologies</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A collaborative annotation project where undergrad...</td>
</tr>
<tr>
<td><a href="../resources/go-central/">infores:go-central</a></td>
<td>GO Central</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The core GO Consortium annotation team responsible...</td>
</tr>
<tr>
<td><a href="../resources/goc/">infores:goc</a></td>
<td>Gene Ontology Consortium</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The consortium that develops and maintains the Gen...</td>
</tr>
<tr>
<td><a href="../resources/disprot/">infores:disprot</a></td>
<td>DisProt</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A database of intrinsically disordered proteins th...</td>
</tr>
<tr>
<td><a href="../resources/agrkb/">infores:agrkb</a></td>
<td>Alliance of Genome Resources (AGR Knowledgebase)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/aact/">infores:aact</a></td>
<td>Aggregate Analysis of ClinicalTrial.gov (AACT) database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>https://ctti-clinicaltrials.org/our-work/quality/s...</td>
</tr>
<tr>
<td><a href="../resources/answer-coalesce/">infores:answer-coalesce</a></td>
<td>Answer coalesce</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>Answer coalesce: This service accepts a TRAPI 1.1 ...</td>
</tr>
<tr>
<td><a href="../resources/aragorn/">infores:aragorn</a></td>
<td>ARAGORN</td>
<td>released</td>
<td>prediction</td>
<td>ARAGORN: Performs a query operation which compiles...</td>
</tr>
<tr>
<td><a href="../resources/aragorn-ranker/">infores:aragorn-ranker</a></td>
<td>ARAGORN Ranker</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>ARAGORN Ranker: The ranker used by the ARAGORN ARA...</td>
</tr>
<tr>
<td><a href="../resources/arax/">infores:arax</a></td>
<td>ARAX Translator Reasoner</td>
<td>released</td>
<td>prediction</td>
<td>TRAPI 1.1 endpoint for the NCATS Biomedical Transl...</td>
</tr>
<tr>
<td><a href="../resources/atc-codes-umls/">infores:atc-codes-umls</a></td>
<td>Anatomical Therapeutic Chemical (ATC) Codes (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/atgo/">infores:atgo</a></td>
<td>Autophagy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/athena/">infores:athena</a></td>
<td>Athena</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-genome-alliance/">infores:automat-genome-alliance</a></td>
<td>Automat Alliance of Genome Resources</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-renci-sri-reference-kg/">infores:automat-renci-sri-reference-kg</a></td>
<td>Automat RENCI SRI Reference KG</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph based on the [Monarch API (https://api.mon...</td>
</tr>
<tr>
<td><a href="../resources/automat-chem-norm/">infores:automat-chem-norm</a></td>
<td>Automat Chemical normalization (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph  linking together similar compounds.  Two ...</td>
</tr>
<tr>
<td><a href="../resources/automat-chembio/">infores:automat-chembio</a></td>
<td>Automat Chembio</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph based on [chem2bio2rdf (http://cheminfov.i...</td>
</tr>
<tr>
<td><a href="../resources/automat-cord19/">infores:automat-cord19</a></td>
<td>Automat Cord19</td>
<td>deprecated</td>
<td>not_provided</td>
<td>A literature co-occurence graph based on parsing t...</td>
</tr>
<tr>
<td><a href="../resources/automat-cord19-scibite/">infores:automat-cord19-scibite</a></td>
<td>Automat Cord19 Scibite (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>not_provided</td>
<td>A literature co-occurence graph based on the [scib...</td>
</tr>
<tr>
<td><a href="../resources/automat-cord19-scigraph/">infores:automat-cord19-scigraph</a></td>
<td>Automat Cord19 Scigraph (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>not_provided</td>
<td>A literature co-occurence graph based on scigraph ...</td>
</tr>
<tr>
<td><a href="../resources/automat-covid-phenotypes/">infores:automat-covid-phenotypes</a></td>
<td>Automat Covid Phenotypes (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A hand-knowledge_assertion graph of phenotypes of ...</td>
</tr>
<tr>
<td><a href="../resources/automat-covidkop/">infores:automat-covidkop</a></td>
<td>Automat Covidkop KG (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>Knowledge graph constructed from Robokop KG by inc...</td>
</tr>
<tr>
<td><a href="../resources/automat-cam-kp/">infores:automat-cam-kp</a></td>
<td>Automat CAM-KP</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>TRAPI interface to database of Causal Activity Mod...</td>
</tr>
<tr>
<td><a href="../resources/automat-cebs/">infores:automat-cebs</a></td>
<td>Automat CEBS</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>CEBS is a public, web-accessible, manually curated...</td>
</tr>
<tr>
<td><a href="../resources/automat-ctd/">infores:automat-ctd</a></td>
<td>Automat CTD</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on the [Comparative Toxicogenomics D...</td>
</tr>
<tr>
<td><a href="../resources/automat-drug-central/">infores:automat-drug-central</a></td>
<td>Automat DrugCentral</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on DrugCentral</td>
</tr>
<tr>
<td><a href="../resources/automat-foodb/">infores:automat-foodb</a></td>
<td>Automat Foodb</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph of the food/chemical relationships in [foo...</td>
</tr>
<tr>
<td><a href="../resources/automat-gtex/">infores:automat-gtex</a></td>
<td>Automat GTEx</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph containing eqtl and sqtl information from ...</td>
</tr>
<tr>
<td><a href="../resources/automat-gtopdb/">infores:automat-gtopdb</a></td>
<td>Automat GtoPdb</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on the [IUPHAR Guide to Pharmacology...</td>
</tr>
<tr>
<td><a href="../resources/automat-gwas-catalog/">infores:automat-gwas-catalog</a></td>
<td>Automat GWAS Catalog</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on the GWAS Catalog</td>
</tr>
<tr>
<td><a href="../resources/automat-hetio/">infores:automat-hetio</a></td>
<td>Automat Hetio</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph based on <a href="het.io">hetionet (het.io)</a>.</td>
</tr>
<tr>
<td><a href="../resources/automat-hetionet/">infores:automat-hetionet</a></td>
<td>Automat Hetionet</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on <a href="het.io">hetionet (het.io)</a>.</td>
</tr>
<tr>
<td><a href="../resources/automat-hgnc/">infores:automat-hgnc</a></td>
<td>Automat HGNC</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Gene families from [HGNC (genenames.org)](genename...</td>
</tr>
<tr>
<td><a href="../resources/automat-hmdb/">infores:automat-hmdb</a></td>
<td>Automat HMDB</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on the [Human Metabolome DataBase (h...</td>
</tr>
<tr>
<td><a href="../resources/automat-human-goa/">infores:automat-human-goa</a></td>
<td>Automat Human GOA</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Human Gene Ontology Annotations from the GO consor...</td>
</tr>
<tr>
<td><a href="../resources/automat-icees-kg/">infores:automat-icees-kg</a></td>
<td>Automat ICEES KG</td>
<td>released</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-intact/">infores:automat-intact</a></td>
<td>Automat IntAct</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Molecular (Gene-Gene) interactions from EBI IntAct...</td>
</tr>
<tr>
<td><a href="../resources/automat-kegg/">infores:automat-kegg</a></td>
<td>Automat KEGG</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph based on the [Kyoto Encyclopedia of Genes ...</td>
</tr>
<tr>
<td><a href="../resources/automat-monarchinitiative/">infores:automat-monarchinitiative</a></td>
<td>Automat Monarch Initiative</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Monarch Knowledge Graph is a reference impleme...</td>
</tr>
<tr>
<td><a href="../resources/automat-mychem-info/">infores:automat-mychem-info</a></td>
<td>Automat MyChem (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph integrating DrugBank, DrugCentral, and Aeo...</td>
</tr>
<tr>
<td><a href="../resources/automat-ontology-hierarchy/">infores:automat-ontology-hierarchy</a></td>
<td>Automat Ontological Hierarchy (trapi v-1.1.0)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph for ontological hierarchy extracted from U...</td>
</tr>
<tr>
<td><a href="../resources/automat-openhealthdata-carolina/">infores:automat-openhealthdata-carolina</a></td>
<td>Automat Openhealth Data at Carolina</td>
<td>released</td>
<td>statistical_association</td>
<td>A graph based on infores:openhealthdata-carolina, ...</td>
</tr>
<tr>
<td><a href="../resources/automat-panther/">infores:automat-panther</a></td>
<td>Automat PANTHER</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph of gene families and pathways from [Panthe...</td>
</tr>
<tr>
<td><a href="../resources/automat-pharos/">infores:automat-pharos</a></td>
<td>Automat Pharos</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph based on [Pharos (pharos.nih.gov)](pharos....</td>
</tr>
<tr>
<td><a href="../resources/automat-robokop-kg/">infores:automat-robokop-kg</a></td>
<td>Automat ROBOKOP KG</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Biomedical Knowledge graph build by constructing r...</td>
</tr>
<tr>
<td><a href="../resources/automat-text-mining-provider/">infores:automat-text-mining-provider</a></td>
<td>Automat Textmining KP</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A literature co-occurrence graph created by the Tr...</td>
</tr>
<tr>
<td><a href="../resources/automat-string-db/">infores:automat-string-db</a></td>
<td>Automat STRING</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Search Tool for the Retrieval of INteracting G...</td>
</tr>
<tr>
<td><a href="../resources/automat-sri-reference-kg/">infores:automat-sri-reference-kg</a></td>
<td>Automat SRI Reference Knowledge Graph</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>The SRI reference KG aims to aggregate knowledge s...</td>
</tr>
<tr>
<td><a href="../resources/automat-ubergraph/">infores:automat-ubergraph</a></td>
<td>Automat Ubergraph</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph representation of Ubergraph, an integratio...</td>
</tr>
<tr>
<td><a href="../resources/automat-ubergraph-nonredundant/">infores:automat-ubergraph-nonredundant</a></td>
<td>Automat Ubergraph Nonredundant</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>A graph representation of Ubergraph, an integratio...</td>
</tr>
<tr>
<td><a href="../resources/automat-viral-proteome/">infores:automat-viral-proteome</a></td>
<td>Automat Viral Proteome</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph consisting of viral proteins from UniProt,...</td>
</tr>
<tr>
<td><a href="../resources/automat-ehr-may-treat-kp/">infores:automat-ehr-may-treat-kp</a></td>
<td>Automat Multiomics EHRMLA May Treat API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph consisting of May Treat associations from ...</td>
</tr>
<tr>
<td><a href="../resources/automat-ehr-clinical-connections-kp/">infores:automat-ehr-clinical-connections-kp</a></td>
<td>Automat Multiomics EHRMLA Clinical Connections API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph consisting of clinical connections associa...</td>
</tr>
<tr>
<td><a href="../resources/bfo/">infores:bfo</a></td>
<td>Basic Formal Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/bgee/">infores:bgee</a></td>
<td>Bgee</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/bigclam/">infores:bigclam</a></td>
<td>Big Cell Line Association Miner</td>
<td>deprecated</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/bigg-models/">infores:bigg-models</a></td>
<td>BiGG Models</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>a knowledgebase of genome-scale metabolic network ...</td>
</tr>
<tr>
<td><a href="../resources/biggim/">infores:biggim</a></td>
<td>Big Gene Interaction Miner</td>
<td>deprecated</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/bindingdb/">infores:bindingdb</a></td>
<td>BindingDB</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>BindingDB is a public, web-accessible database of ...</td>
</tr>
<tr>
<td><a href="../resources/bio2rdf/">infores:bio2rdf</a></td>
<td>Bio2RDF</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biocatalogue/">infores:biocatalogue</a></td>
<td>BioCatalogue</td>
<td>deprecated</td>
<td>other</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biogrid/">infores:biogrid</a></td>
<td>The Biological General Repository for Interaction Datasets</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biolink-api/">infores:biolink-api</a></td>
<td>BioLink API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>API integration layer for linked biological object...</td>
</tr>
<tr>
<td><a href="../resources/biolink-model-lookup/">infores:biolink-model-lookup</a></td>
<td>Biolink Model Lookup</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Biolink Model Lookup service.</td>
</tr>
<tr>
<td><a href="../resources/biolink-ontology/">infores:biolink-ontology</a></td>
<td>Biolink ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/bioportal/">infores:bioportal</a></td>
<td>National Center for Biomedical Ontology BioPortal</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-biggim-1/">infores:biothings-biggim-1</a></td>
<td>Big GIM 1 API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>Documentation of the Big GIM 1 KP query web servic...</td>
</tr>
<tr>
<td><a href="../resources/biothings-dgidb/">infores:biothings-dgidb</a></td>
<td>BioThings DGIdb API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the BioThings DGIdb query web ser...</td>
</tr>
<tr>
<td><a href="../resources/biothings-diseases/">infores:biothings-diseases</a></td>
<td>BioThings DISEASES API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the DISEASES query web services.</td>
</tr>
<tr>
<td><a href="../resources/biothings-ebi-gene2phenotype/">infores:biothings-ebi-gene2phenotype</a></td>
<td>BioThings EBIgene2phenotype API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the EBIgene2phenotype query web s...</td>
</tr>
<tr>
<td><a href="../resources/biothings-explorer/">infores:biothings-explorer</a></td>
<td>BioThings Explorer</td>
<td>released</td>
<td>prediction</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/service-provider-trapi/">infores:service-provider-trapi</a></td>
<td>Service Provider TRAPI endpoints</td>
<td>released</td>
<td>other</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-go-bp/">infores:biothings-go-bp</a></td>
<td>BioThings Gene Ontology Biological Process API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the Gene Ontology Biological Proc...</td>
</tr>
<tr>
<td><a href="../resources/biothings-go-cc/">infores:biothings-go-cc</a></td>
<td>BioThings Gene Ontology Cellular Component API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the Gene Ontology Cellular Compon...</td>
</tr>
<tr>
<td><a href="../resources/biothings-go-mf/">infores:biothings-go-mf</a></td>
<td>BioThings Gene Ontology Molecular Activity API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the Gene Ontology Molecular Activ...</td>
</tr>
<tr>
<td><a href="../resources/biothings-hpo/">infores:biothings-hpo</a></td>
<td>BioThings Human Phenotype Ontology API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the HPO query web services.</td>
</tr>
<tr>
<td><a href="../resources/biothings-idisk/">infores:biothings-idisk</a></td>
<td>BioThings iDISK API</td>
<td>released</td>
<td>mixed</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-mgi-g2p/">infores:biothings-mgi-g2p</a></td>
<td>BioThings MGIgene2phenotype API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the MGIgene2phenotype query web s...</td>
</tr>
<tr>
<td><a href="../resources/biothings-multiomics-biggim-drugresponse/">infores:biothings-multiomics-biggim-drugresponse</a></td>
<td>Multiomics BigGIM-DrugResponse KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the Drug Response KP query web se...</td>
</tr>
<tr>
<td><a href="../resources/biothings-multiomics-ehr-risk/">infores:biothings-multiomics-ehr-risk</a></td>
<td>Multiomics EHR Risk KP API</td>
<td>modified</td>
<td>statistical_association</td>
<td>Documentation of the Multiomics Electronic-Health-...</td>
</tr>
<tr>
<td><a href="../resources/catrax-pharmacogenomics/">infores:catrax-pharmacogenomics</a></td>
<td>Multiomics Pharmacogenomics KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the Pharmacogenomics KP query web...</td>
</tr>
<tr>
<td><a href="../resources/providence-st-joseph-ehr/">infores:providence-st-joseph-ehr</a></td>
<td>Providence St. Joseph EHR Data</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A partnership with Providence/Swedish Health Servi...</td>
</tr>
<tr>
<td><a href="../resources/isb-EHRMLA-data/">infores:isb-EHRMLA-data</a></td>
<td>Multiomics EHRMLA May Treat API</td>
<td>released</td>
<td>prediction</td>
<td>Documentation of the Multiomics Electronic-Health-...</td>
</tr>
<tr>
<td><a href="../resources/isb-EHRMLA-clinicalconnections/">infores:isb-EHRMLA-clinicalconnections</a></td>
<td>Multiomics EHRMLA Clinical Connections API</td>
<td>released</td>
<td>prediction</td>
<td>Documentation of the Multiomics Electronic-Health-...</td>
</tr>
<tr>
<td><a href="../resources/biothings-multiomics-wellness/">infores:biothings-multiomics-wellness</a></td>
<td>Multiomics Wellness KP API</td>
<td>released</td>
<td>statistical_association</td>
<td>Documentation of the Multiomics Wellness KP query ...</td>
</tr>
<tr>
<td><a href="../resources/multiomics-clinicaltrials/">infores:multiomics-clinicaltrials</a></td>
<td>Multiomics ClinicalTrials KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-multiomics-clinicaltrials/">infores:biothings-multiomics-clinicaltrials</a></td>
<td>Multiomics ClinicalTrials KP API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/multiomics-drugapprovals/">infores:multiomics-drugapprovals</a></td>
<td>Multiomics Drug Approvals KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/multiomics-multiomics/">infores:multiomics-multiomics</a></td>
<td>Multiomics KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/multiomics-microbiome/">infores:multiomics-microbiome</a></td>
<td>Multiomics Microbiome KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-pfocr/">infores:biothings-pfocr</a></td>
<td>BioThings PFOCR API</td>
<td>modified</td>
<td>prediction</td>
<td>Documentation of the pfocr query web services.</td>
</tr>
<tr>
<td><a href="../resources/pfocr/">infores:pfocr</a></td>
<td>PFOCR</td>
<td>released</td>
<td>prediction</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb/">infores:biothings-semmeddb</a></td>
<td>BioThings SEMMEDDB API</td>
<td>released</td>
<td>not_provided</td>
<td>Documentation of the BioThings SEMMEDDB query web ...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-anatomy/">infores:biothings-semmeddb-anatomy</a></td>
<td>SEMMED Anatomy API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-biological-process/">infores:biothings-semmeddb-biological-process</a></td>
<td>SEMMED Biological Process API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-chemical/">infores:biothings-semmeddb-chemical</a></td>
<td>SEMMED Chemical API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-disease/">infores:biothings-semmeddb-disease</a></td>
<td>SEMMED Disease API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-gene/">infores:biothings-semmeddb-gene</a></td>
<td>SEMMED Gene API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-semmeddb-phenotype/">infores:biothings-semmeddb-phenotype</a></td>
<td>SEMMED Phenotype API</td>
<td>deprecated</td>
<td>not_provided</td>
<td>Documentation of the SEMMED disease query web serv...</td>
</tr>
<tr>
<td><a href="../resources/biothings-tcga-mut-freq/">infores:biothings-tcga-mut-freq</a></td>
<td>Multiomics TCGA Mutation Frequency KP API</td>
<td>deprecated</td>
<td>statistical_association</td>
<td>Documentation of the TCGA Mutation Frequency KP qu...</td>
</tr>
<tr>
<td><a href="../resources/biothings-uberon-ontology/">infores:biothings-uberon-ontology</a></td>
<td>BioThings UBERON Ontology API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the UBERON query web services.</td>
</tr>
<tr>
<td><a href="../resources/biothings-rhea/">infores:biothings-rhea</a></td>
<td>BioThings Rhea API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings Rhea query web serv...</td>
</tr>
<tr>
<td><a href="../resources/rhea/">infores:rhea</a></td>
<td>Rhea</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>https://www.rhea-db.org/</td>
</tr>
<tr>
<td><a href="../resources/biothings-bioplanet-pathway-disease/">infores:biothings-bioplanet-pathway-disease</a></td>
<td>BioThings BioPlanet Pathway-Disease API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings [BioPlanet](https:/...</td>
</tr>
<tr>
<td><a href="../resources/biothings-bioplanet-pathway-gene/">infores:biothings-bioplanet-pathway-gene</a></td>
<td>BioThings BioPlanet Pathway-Gene API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings [BioPlanet](https:/...</td>
</tr>
<tr>
<td><a href="../resources/bioplanet/">infores:bioplanet</a></td>
<td>BioPlanet</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>https://tripod.nih.gov/bioplanet/#</td>
</tr>
<tr>
<td><a href="../resources/biothings-bindingdb/">infores:biothings-bindingdb</a></td>
<td>BioThings BindingDB API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the BioThings [BindingDB](https:/...</td>
</tr>
<tr>
<td><a href="../resources/bspo/">infores:bspo</a></td>
<td>Biological Spatial Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/dailymed/">infores:dailymed</a></td>
<td>DailyMed</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The DailyMed database contains labeling, submitted...</td>
</tr>
<tr>
<td><a href="../resources/medi/">infores:medi</a></td>
<td>MeDI</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Indications and contraindications from DailyMed, e...</td>
</tr>
<tr>
<td><a href="../resources/wbbt/">infores:wbbt</a></td>
<td>C. elegans Gross Anatomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of the anatomy ...</td>
</tr>
<tr>
<td><a href="../resources/wbphenotype/">infores:wbphenotype</a></td>
<td>C. elegans phenotype</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of Caenorhabdit...</td>
</tr>
<tr>
<td><a href="../resources/wbls/">infores:wbls</a></td>
<td>C. elegans development ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of the developm...</td>
</tr>
<tr>
<td><a href="../resources/cam-kp/">infores:cam-kp</a></td>
<td>CAM-KP API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>TRAPI interface to database of Causal Activity Mod...</td>
</tr>
<tr>
<td><a href="../resources/campfhir/">infores:campfhir</a></td>
<td>Clinical Asset Mapping Program for FHIR</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cebs/">infores:cebs</a></td>
<td>CEBS</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>CEBS is a public, web-accessible, manually curated...</td>
</tr>
<tr>
<td><a href="../resources/cellmarker/">infores:cellmarker</a></td>
<td>CellMarker 2.0</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/chebi/">infores:chebi</a></td>
<td>Chemical Entity of Biological Interest</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>a freely available dictionary of molecular entitie...</td>
</tr>
<tr>
<td><a href="../resources/chem2bio2rdf/">infores:chem2bio2rdf</a></td>
<td>Chem2bio2RDF</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/chembank/">infores:chembank</a></td>
<td>ChemBank</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>Initiative for Chemical Genetics contract from the...</td>
</tr>
<tr>
<td><a href="../resources/chembio/">infores:chembio</a></td>
<td>ChemBio</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/chembl/">infores:chembl</a></td>
<td>ChEMBL</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>a manually knowledge_assertion database of bioacti...</td>
</tr>
<tr>
<td><a href="../resources/chemotext/">infores:chemotext</a></td>
<td>Chemotext</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/civic/">infores:civic</a></td>
<td>Clinital Interpretation of Variants in Cancer</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cl/">infores:cl</a></td>
<td>Cell Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/clingen/">infores:clingen</a></td>
<td>Clinical Genome Resource (ClinGen)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>ClinGen is a NIH-funded resource dedicated to buil...</td>
</tr>
<tr>
<td><a href="../resources/clinical-profiles/">infores:clinical-profiles</a></td>
<td>Clinical Profiles</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/clinicaltrials/">infores:clinicaltrials</a></td>
<td>ClinicalTrials.gov</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/clinvar/">infores:clinvar</a></td>
<td>ClinVar</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>ClinVar is a freely accessible, public archive of ...</td>
</tr>
<tr>
<td><a href="../resources/cmap/">infores:cmap</a></td>
<td>Connectivity Map</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>genome-scale library of cellular signatures that c...</td>
</tr>
<tr>
<td><a href="../resources/columbia-cdw-ehr-data/">infores:columbia-cdw-ehr-data</a></td>
<td>Columbia Clinical Data Warehouse for Health Patient EHR Data</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Columbia Clinical Data Warehouse (CDW) contain...</td>
</tr>
<tr>
<td><a href="../resources/cohd/">infores:cohd</a></td>
<td>Columbia Open Health Data (COHD)</td>
<td>released</td>
<td>statistical_association</td>
<td>The Columbia Open Health Data (COHD) API provides ...</td>
</tr>
<tr>
<td><a href="../resources/cohd-covid/">infores:cohd-covid</a></td>
<td>Columbia Open Health Data (COHD) for COVID-19 Research</td>
<td>released</td>
<td>statistical_association</td>
<td>The Columbia Open Health Data (COHD) for COVID-19 ...</td>
</tr>
<tr>
<td><a href="../resources/cohd-covid19-api/">infores:cohd-covid19-api</a></td>
<td>Columbia Open Health Data for COVID-19 Research API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/connections-hypothesis/">infores:connections-hypothesis</a></td>
<td>Connections Hypothesis Provider API</td>
<td>released</td>
<td>prediction</td>
<td>A Translator Reasoner API for the Connections Hypo...</td>
</tr>
<tr>
<td><a href="../resources/cord19/">infores:cord19</a></td>
<td>COVID-19 Open Research Dataset</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cord19-scibite/">infores:cord19-scibite</a></td>
<td>COVID-19 Open Research Dataset Scibite</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cosmic/">infores:cosmic</a></td>
<td>Catalog of Somatic Mutations in Cancer</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/covid-phenotypes/">infores:covid-phenotypes</a></td>
<td>Covid Phenotypes</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cpdb/">infores:cpdb</a></td>
<td>ConsensusPathDB</td>
<td>released</td>
<td>mixed</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cpt-codes-umls/">infores:cpt-codes-umls</a></td>
<td>Current Procedural Terminology (CPT) Codes (from UMLS)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/cqs/">infores:cqs</a></td>
<td>Translator knowledge_assertion Query Service</td>
<td>released</td>
<td>prediction</td>
<td>The Translator Curated Query Service (CQS) is an S...</td>
</tr>
<tr>
<td><a href="../resources/ctd/">infores:ctd</a></td>
<td>Comparative Toxicogenomics Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A robust, publicly available database that aims to...</td>
</tr>
<tr>
<td><a href="../resources/ctrp/">infores:ctrp</a></td>
<td>Cancer Therapeutics Response Portal</td>
<td>released</td>
<td>statistical_association</td>
<td>links genetic, lineage, and other cellular feature...</td>
</tr>
<tr>
<td><a href="../resources/date/">infores:date</a></td>
<td>Drugs to target pAthways by the Tissue Expression</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/dbsnp/">infores:dbsnp</a></td>
<td>Single Nucleotide Polymorphism Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/dda/">infores:dda</a></td>
<td>Dictyostelium discoideum anatomy</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ddpheno/">infores:ddpheno</a></td>
<td>Dictyostelium discoideum phenotype ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of phenotypes o...</td>
</tr>
<tr>
<td><a href="../resources/delta-qt-db/">infores:delta-qt-db</a></td>
<td>delta QT Database</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/depmap/">infores:depmap</a></td>
<td>Cancer Dependency Map</td>
<td>released</td>
<td>statistical_association</td>
<td>Cancer Dependency Map to systematically identify g...</td>
</tr>
<tr>
<td><a href="../resources/dgidb/">infores:dgidb</a></td>
<td>Drug Gene Interaction Database</td>
<td>released</td>
<td>mixed</td>
<td>information on drug-gene interactions and druggabl...</td>
</tr>
<tr>
<td><a href="../resources/dili-network-study-data/">infores:dili-network-study-data</a></td>
<td>Drug-Induced Liver Injury Network (DILIN) Participant Data</td>
<td>deprecated</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/faers/">infores:faers</a></td>
<td>FDA Adverse Event Reporting System</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The FDA Adverse Event Reporting System (FAERS) is ...</td>
</tr>
<tr>
<td><a href="../resources/fbbt/">infores:fbbt</a></td>
<td>Drosophila gross anatomy</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An ontology representing the gross anatomy of Dros...</td>
</tr>
<tr>
<td><a href="../resources/fbcv/">infores:fbcv</a></td>
<td>FlyBase Controlled Vocabulary</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary used for variou...</td>
</tr>
<tr>
<td><a href="../resources/fbdv/">infores:fbdv</a></td>
<td>Drosophila Developmental Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of the developm...</td>
</tr>
<tr>
<td><a href="../resources/disease-ontology/">infores:disease-ontology</a></td>
<td>Disease Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/diseases/">infores:diseases</a></td>
<td>DISEASES</td>
<td>released</td>
<td>mixed</td>
<td>DISEASES is a weekly updated web resource that int...</td>
</tr>
<tr>
<td><a href="../resources/disgenet/">infores:disgenet</a></td>
<td>DisGeNET</td>
<td>released</td>
<td>mixed</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/distild/">infores:distild</a></td>
<td>Diseases &amp; Traits in LD database</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/doaf/">infores:doaf</a></td>
<td>Disease Ontology Annotation Framework (DOAF)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/docm/">infores:docm</a></td>
<td>Database of knowledge_assertion Mutations</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/drug-repurposing-hub/">infores:drug-repurposing-hub</a></td>
<td>Drug Repurposing Hub</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>knowledge_assertion and annotated collection of FD...</td>
</tr>
<tr>
<td><a href="../resources/drugbank/">infores:drugbank</a></td>
<td>DrugBank</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A comprehensive, free-to-access, online database c...</td>
</tr>
<tr>
<td><a href="../resources/drugcentral/">infores:drugcentral</a></td>
<td>DrugCentral</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Online drug information resource created and maint...</td>
</tr>
<tr>
<td><a href="../resources/drugmechdb/">infores:drugmechdb</a></td>
<td>DrugMechDB</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A database of paths that represent the mechanism o...</td>
</tr>
<tr>
<td><a href="../resources/ebi/">infores:ebi</a></td>
<td>European Bioinformatics Institute</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ebi-gene2phenotype/">infores:ebi-gene2phenotype</a></td>
<td>European Bioinformatics Institute Gene to Phenotype Resource</td>
<td>modified</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ebi-proteins/">infores:ebi-proteins</a></td>
<td>EBI Proteins API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Proteins REST API provides access to key biolo...</td>
</tr>
<tr>
<td><a href="../resources/ebi-quick-go/">infores:ebi-quick-go</a></td>
<td>QuickGO API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The QuickGO REST API provides access to key biolog...</td>
</tr>
<tr>
<td><a href="../resources/eco/">infores:eco</a></td>
<td>Evidence &amp; Conclusion Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ecto/">infores:ecto</a></td>
<td>Environmental conditions, treatments and exposures ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/eram/">infores:eram</a></td>
<td>eRAM: encyclopedia of rare disease annotations for precision medicine</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>encyclopedia of rare disease annotations for preci...</td>
</tr>
<tr>
<td><a href="../resources/efo/">infores:efo</a></td>
<td>Experimental Factor Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ehdaa2/">infores:ehdaa2</a></td>
<td>Human Developmental Anatomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hsapdv/">infores:hsapdv</a></td>
<td>Human Developmental Stages Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An ontology that covers life cycle stages for huma...</td>
</tr>
<tr>
<td><a href="../resources/ensembl-gene/">infores:ensembl-gene</a></td>
<td>Ensembl gene</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/entrez/">infores:entrez</a></td>
<td>Entrez</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/epsd/">infores:epsd</a></td>
<td>EPSD</td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/erc-analysis/">infores:erc-analysis</a></td>
<td>Evolutionary Rate Covariation (ERC) Analysis</td>
<td>deprecated</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/explanatory-agent/">infores:explanatory-agent</a></td>
<td>Explanatory Agent API</td>
<td>deprecated</td>
<td>prediction</td>
<td>A Translator Reasoner API for the Explanatory Agen...</td>
</tr>
<tr>
<td><a href="../resources/fhir-pit/">infores:fhir-pit</a></td>
<td>FHIR Patient data Integration Tool</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/fma-obo/">infores:fma-obo</a></td>
<td>Foundational Model of Anatomy Ontology (FMA -- both from UMLS and from OBO)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/fma-umls/">infores:fma-umls</a></td>
<td>Foundational Model of Anatomy Ontology (FMA -- both from UMLS and from OBO)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/foodb/">infores:foodb</a></td>
<td>Food Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/foodon/">infores:foodon</a></td>
<td>Food Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gdc/">infores:gdc</a></td>
<td>National Cancer Institute Genomic Data Commons Data Portal</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gdsc/">infores:gdsc</a></td>
<td>Genomics of Drug Sensitivity in Cancer</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/genage/">infores:genage</a></td>
<td>GenAge Database of Ageing-Related Genes</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gencc/">infores:gencc</a></td>
<td>The Gene Curation Coalition (GenCC)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The GenCC DB provides information pertaining to th...</td>
</tr>
<tr>
<td><a href="../resources/gendr/">infores:gendr</a></td>
<td>GenDR Database of Dietary Restriction-Related Genes</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/genebass/">infores:genebass</a></td>
<td>Genebass: Gene-based association summary statistics</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Genebass is a resource of exome-based association ...</td>
</tr>
<tr>
<td><a href="../resources/genepio/">infores:genepio</a></td>
<td>Genomic Epidemiology Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/geneprof/">infores:geneprof</a></td>
<td>GeneProf</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>underlying resource is throwing 404 error atm</td>
</tr>
<tr>
<td><a href="../resources/genetics-data-provider/">infores:genetics-data-provider</a></td>
<td>Genetics KP</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>TRAPI 1.1 endpoint for the NCATS Biomedical Transl...</td>
</tr>
<tr>
<td><a href="../resources/ghr/">infores:ghr</a></td>
<td>Genetics Home Reference</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gnbr/">infores:gnbr</a></td>
<td>Global Network of Biomedical Relationships</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/go/">infores:go</a></td>
<td>Gene Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/go-cam/">infores:go-cam</a></td>
<td>Gene Ontology Causal Activity Model Annotations</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/aop-cam/">infores:aop-cam</a></td>
<td>Adverse Outcome Pathways Casual Activity Models</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/go-plus/">infores:go-plus</a></td>
<td>Gene Ontology Plus</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/human-goa/">infores:human-goa</a></td>
<td>Human Gene Ontology Annotations</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/goa/">infores:goa</a></td>
<td>Gene Ontology Annotations</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gote/">infores:gote</a></td>
<td>GOTE (Gpcrs to dOwnstream cellular pathways byTissue Expression)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gtrx/">infores:gtrx</a></td>
<td>Genome-to-Treatment</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gtex/">infores:gtex</a></td>
<td>Genotype-Tissue Expression</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/gtopdb/">infores:gtopdb</a></td>
<td>Guide to Pharmacology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An expert-knowledge_assertion resource of ligand-a...</td>
</tr>
<tr>
<td><a href="../resources/gwas-catalog/">infores:gwas-catalog</a></td>
<td>GWAS Catalog</td>
<td>released</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hcp-codes-umls/">infores:hcp-codes-umls</a></td>
<td>Healthcare Common Procedure Coding System (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hcpcs-cpt-umls/">infores:hcpcs-cpt-umls</a></td>
<td>HCPCS Version of Current Procedural Terminology (HCPT) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hetionet/">infores:hetionet</a></td>
<td>Hetionet</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hgnc/">infores:hgnc</a></td>
<td>HUGO Gene Nomenclature Committee</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hl7-umls/">infores:hl7-umls</a></td>
<td>Health Level Seven (HL7) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hmdb/">infores:hmdb</a></td>
<td>Human Metabolome Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A freely available electronic database containing ...</td>
</tr>
<tr>
<td><a href="../resources/homologene/">infores:homologene</a></td>
<td>HomoloGene</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hopkins-synthetic-patient-data/">infores:hopkins-synthetic-patient-data</a></td>
<td>Hopkins Synthetic Patient Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hpa/">infores:hpa</a></td>
<td>Human Protein Atlas</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Human Protein Atlas is a Swedish-based program...</td>
</tr>
<tr>
<td><a href="../resources/hpo/">infores:hpo</a></td>
<td>Human Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/hpo-annotations/">infores:hpo-annotations</a></td>
<td>Human Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd10cm/">infores:icd10cm</a></td>
<td>ICD10 Clinical Modification</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd11-foundation/">infores:icd11-foundation</a></td>
<td>ICD11 Foundation</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd10-umls/">infores:icd10-umls</a></td>
<td>ICD10 (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd10ae-umls/">infores:icd10ae-umls</a></td>
<td>ICD10AE (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd10cm-umls/">infores:icd10cm-umls</a></td>
<td>ICD10CM (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd10pcs-umls/">infores:icd10pcs-umls</a></td>
<td>ICD10PCS (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icd9cm-umls/">infores:icd9cm-umls</a></td>
<td>ICD9CM (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/icees-kg/">infores:icees-kg</a></td>
<td>ICEES (Integrated Clinical and Environmental Exposures Service)</td>
<td>released</td>
<td>statistical_association</td>
<td>ICEES Knowledge Graph (KG) is an open service that...</td>
</tr>
<tr>
<td><a href="../resources/idisk/">infores:idisk</a></td>
<td>integrated Dietary Supplement Knowledge Base</td>
<td>released</td>
<td>mixed</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC70755...</td>
</tr>
<tr>
<td><a href="../resources/improving-agent/">infores:improving-agent</a></td>
<td>imProving Agent</td>
<td>released</td>
<td>prediction</td>
<td>imProving Agent OpenAPI TRAPI Specification</td>
</tr>
<tr>
<td><a href="../resources/indigo-reasoner/">infores:indigo-reasoner</a></td>
<td>OpenAPI for indigo NCATS Biomedical Translator Reasoner</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>OpenAPI for indigo NCATS Biomedical Translator Rea...</td>
</tr>
<tr>
<td><a href="../resources/ino/">infores:ino</a></td>
<td>Interaction Network Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/intact/">infores:intact</a></td>
<td>IntAct</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/interactome-ccsb/">infores:interactome-ccsb</a></td>
<td>Interactome Projects at CCSB</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/interpro/">infores:interpro</a></td>
<td>InterPro</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/inxight-drugs/">infores:inxight-drugs</a></td>
<td>Inxight: Drugs</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>NCATS Inxight Drugs contains information on ingred...</td>
</tr>
<tr>
<td><a href="../resources/iproclass/">infores:iproclass</a></td>
<td>iProClass</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/iptm-net/">infores:iptm-net</a></td>
<td>iPTMNet</td>
<td>released</td>
<td>not_provided</td>
<td>A bioinformatics resource for integrated understan...</td>
</tr>
<tr>
<td><a href="../resources/irefindex/">infores:irefindex</a></td>
<td>iRefIndex</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/isb-incov/">infores:isb-incov</a></td>
<td>Institute for Systems Biology COVID-19 Immune Response Study</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/isb-wellness/">infores:isb-wellness</a></td>
<td>Institute for Systems Biology Scientific Wellness</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/kegg/">infores:kegg</a></td>
<td>Kyoto Encyclopedia of Genes and Genomes (KEGG)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/kgea/">infores:kgea</a></td>
<td>Knowledge Graph Exchange Archive</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/kinace/">infores:kinace</a></td>
<td>KinAce</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>https://kinace.kinametrix.com/#section-about The K...</td>
</tr>
<tr>
<td><a href="../resources/kinomescan/">infores:kinomescan</a></td>
<td>KINOMEscan</td>
<td>released</td>
<td>other</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/knowledge-collaboratory/">infores:knowledge-collaboratory</a></td>
<td>Translator Knowledge Collaboratory API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Translator Reasoner API for the Knowledge Collabor...</td>
</tr>
<tr>
<td><a href="../resources/life-science-resource-registry/">infores:life-science-resource-registry</a></td>
<td>Life Science Resource Registry</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/lincs/">infores:lincs</a></td>
<td>Library of Integrated Network-Based Cellular Signatures</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/linkedspl/">infores:linkedspl</a></td>
<td>Linked Structured Product Labels</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/litvar/">infores:litvar</a></td>
<td>LitVar API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>LitVar allows the search and retrieval of variant ...</td>
</tr>
<tr>
<td><a href="../resources/loinc/">infores:loinc</a></td>
<td>Logical Observation Identifiers Names and Codes</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/loinc-umls/">infores:loinc-umls</a></td>
<td>LOINC (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mp/">infores:mp</a></td>
<td>Mammalian Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Standard terms for annotating mammalian phenotypic...</td>
</tr>
<tr>
<td><a href="../resources/maxo/">infores:maxo</a></td>
<td>Medical Action Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>see also https://github.com/monarch-initiative/MAx...</td>
</tr>
<tr>
<td><a href="../resources/meddra-umls/">infores:meddra-umls</a></td>
<td>MedDRA (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/medgen/">infores:medgen</a></td>
<td>MedGen</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/medlineplus/">infores:medlineplus</a></td>
<td>MedlinePlus (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/medrt-umls/">infores:medrt-umls</a></td>
<td>Medication Reference Terminology (MED-RT) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mesh/">infores:mesh</a></td>
<td>Medical Subject Headings Thesaurus</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/metacyc/">infores:metacyc</a></td>
<td>MetaCyc Metabolic Pathway Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/emapa/">infores:emapa</a></td>
<td>Mouse Developmental Anatomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An ontology for mouse anatomy covering embryonic d...</td>
</tr>
<tr>
<td><a href="../resources/mgi/">infores:mgi</a></td>
<td>Mouse Genome Informatics</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mpath/">infores:mpath</a></td>
<td>Mouse pathology ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of mutant and t...</td>
</tr>
<tr>
<td><a href="../resources/mi/">infores:mi</a></td>
<td>Molecular Interactions Controlled Vocabulary</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mirbase/">infores:mirbase</a></td>
<td>The microRNA Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mirgate/">infores:mirgate</a></td>
<td>MiRGate</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/molepro/">infores:molepro</a></td>
<td>Molecular Data Provider for NCATS Biomedical Translator Reasoners</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Molecular Data Provider for NCATS Biomedical Trans...</td>
</tr>
<tr>
<td><a href="../resources/monarch-data/">infores:monarch-data</a></td>
<td>SciGraph-Monarch-Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/monarch-ontology/">infores:monarch-ontology</a></td>
<td>SciGraph-Monarch-Ontology</td>
<td>deprecated</td>
<td>other</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/monarchinitiative/">infores:monarchinitiative</a></td>
<td>Monarch Initiative</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mondo/">infores:mondo</a></td>
<td>MONDO Disease Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mop/">infores:mop</a></td>
<td>Molecular Process Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mousemine/">infores:mousemine</a></td>
<td>Mouse Genome Informatics Mousemine</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/msigdb/">infores:msigdb</a></td>
<td>Molecular Signatures Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/mychem-info/">infores:mychem-info</a></td>
<td>MyChem.info API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the MyChem.info chem query web se...</td>
</tr>
<tr>
<td><a href="../resources/mydisease-info/">infores:mydisease-info</a></td>
<td>MyDisease.info API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the MyDisease.info disease query ...</td>
</tr>
<tr>
<td><a href="../resources/mygene-info/">infores:mygene-info</a></td>
<td>MyGene.info API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the MyGene.info Gene Query web se...</td>
</tr>
<tr>
<td><a href="../resources/myvariant-info/">infores:myvariant-info</a></td>
<td>MyVariant.info API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the MyVariant.info Variant Query ...</td>
</tr>
<tr>
<td><a href="../resources/nanopublications/">infores:nanopublications</a></td>
<td>Nanopublications Network</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nbo/">infores:nbo</a></td>
<td>Neuro Behavior Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncats-ars/">infores:ncats-ars</a></td>
<td>National Center for Advancing Translational Science Biomedical Data Translator Autonomous Relay System</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncbi-gene/">infores:ncbi-gene</a></td>
<td>National Center for Biotechnology Information Gene</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncbi-taxon/">infores:ncbi-taxon</a></td>
<td>NCBI Taxonomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncbi-taxonomy/">infores:ncbi-taxonomy</a></td>
<td>NCBI Taxonomy</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncdeq-cafo-exposures-data/">infores:ncdeq-cafo-exposures-data</a></td>
<td>North Carolina Department of Environmental Quality Concentrated Animal Feeding Operations Exposures Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncdeq-landfill-exposures-data/">infores:ncdeq-landfill-exposures-data</a></td>
<td>North Carolina Department of Environmental Quality Landfill Exposures Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nces-schools-exposure-data/">infores:nces-schools-exposure-data</a></td>
<td>NCES public school exposures data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ncit/">infores:ncit</a></td>
<td>National Cancer Institute Thesaurus</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ndcd/">infores:ndcd</a></td>
<td>National Drug Code Directory</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nddf-umls/">infores:nddf-umls</a></td>
<td>National Drug Data File (NDDF) (from UMLS)</td>
<td>deprecated</td>
<td></td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ndex/">infores:ndex</a></td>
<td>NDEx</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ndfrt/">infores:ndfrt</a></td>
<td>National Drug File - Reference Terminology (NDF-RT) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nexo-align/">infores:nexo-align</a></td>
<td>NEXO Align Ontology</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nexo-construct/">infores:nexo-construct</a></td>
<td>NEXO Construct Ontology</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/niehs-epr-study-datae/">infores:niehs-epr-study-datae</a></td>
<td>NIEHS Environmental Polymorphisms Registry</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nord/">infores:nord</a></td>
<td>National Organization for Rare Disorders (NORD) Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/nsides/">infores:nsides</a></td>
<td>nSides</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/openhealthdata-carolina/">infores:openhealthdata-carolina</a></td>
<td>Open Health Data @ Carolina</td>
<td>released</td>
<td>statistical_association</td>
<td>Open Health Data @ Carolina provides access to cou...</td>
</tr>
<tr>
<td><a href="../resources/ols/">infores:ols</a></td>
<td>Ontology Lookup Service API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The OLS REST API provides access to key biological...</td>
</tr>
<tr>
<td><a href="../resources/oba/">infores:oba</a></td>
<td>Ontology of Biological Attributes</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A collection of biological attributes (traits) cov...</td>
</tr>
<tr>
<td><a href="../resources/omicsdi/">infores:omicsdi</a></td>
<td>OmicsDI</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/omim/">infores:omim</a></td>
<td>Online Mendelian Inheritance in Man (OMIM)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/omnicorp/">infores:omnicorp</a></td>
<td>OmniCorp</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/omop-cdm/">infores:omop-cdm</a></td>
<td>Observational Medical Outcomes Partnership Common Data Model</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/omop-ohdsi-api/">infores:omop-ohdsi-api</a></td>
<td>Observational Medical Outcomes Partnership Observational Health Data Sciences and Informatics API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/open-targets/">infores:open-targets</a></td>
<td>Open Targets</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/openpredict/">infores:openpredict</a></td>
<td>OpenPredict API</td>
<td>released</td>
<td>prediction</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ordo/">infores:ordo</a></td>
<td>Orphanet Rare Disease Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/orphanet/">infores:orphanet</a></td>
<td>Orphanet</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/owlsim/">infores:owlsim</a></td>
<td>OWLSim Ontology Based Profile Matching</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/oxo/">infores:oxo</a></td>
<td>Ontology Xref Service</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/panther/">infores:panther</a></td>
<td>Panther Classification System</td>
<td>released</td>
<td>other</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/path-pheno-db/">infores:path-pheno-db</a></td>
<td>PathoPhenoDB</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pathway-commons/">infores:pathway-commons</a></td>
<td>Pathway Commons</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pathwhiz/">infores:pathwhiz</a></td>
<td>PathWhiz</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pato/">infores:pato</a></td>
<td>Phenotype and Trait Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/phenio/">infores:phenio</a></td>
<td>Phenomics Integrated Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An ontology for accessing and comparing knowledge ...</td>
</tr>
<tr>
<td><a href="../resources/patric/">infores:patric</a></td>
<td>Pathosystems Resource Integration Center</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pgxmine/">infores:pgxmine</a></td>
<td>Text Mined pharmacogenomic polymorphisms</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Text mined pharmacogenomic polymorphisms to assist...</td>
</tr>
<tr>
<td><a href="../resources/pdsp/">infores:pdsp</a></td>
<td>Psychoactive Drug Screening Program</td>
<td>released</td>
<td>other</td>
<td>This service provides screening of novel psychoact...</td>
</tr>
<tr>
<td><a href="../resources/pdq-umls/">infores:pdq-umls</a></td>
<td>Physician Data Query (PDQ) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pfam/">infores:pfam</a></td>
<td>Pfam</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Pfam database is a large collection of protein...</td>
</tr>
<tr>
<td><a href="../resources/pharmacotherapydb/">infores:pharmacotherapydb</a></td>
<td>Pharmacotherapy Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pharmgkb/">infores:pharmgkb</a></td>
<td>Pharmacogenomics Knowledgebase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pharos/">infores:pharos</a></td>
<td>Pharos</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>focusing on three of the most commonly drug-target...</td>
</tr>
<tr>
<td><a href="../resources/psite-plus/">infores:psite-plus</a></td>
<td>PhosphoSitePlus</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>provides comprehensive information and tools for t...</td>
</tr>
<tr>
<td><a href="../resources/pr/">infores:pr</a></td>
<td>Protein Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/prefixcommons/">infores:prefixcommons</a></td>
<td>Prefix Commons</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/preppi/">infores:preppi</a></td>
<td>PrePPI</td>
<td>released</td>
<td>prediction</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/primekg/">infores:primekg</a></td>
<td>PrimeKG, Precision Medicine Knowledge Graph</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/probe-miner/">infores:probe-miner</a></td>
<td>Probe Miner</td>
<td>released</td>
<td>statistical_association</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/psy-umls/">infores:psy-umls</a></td>
<td>Psychological Index Terms (PSY) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pubchem/">infores:pubchem</a></td>
<td>PubChem</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>an open chemistry database at the National Institu...</td>
</tr>
<tr>
<td><a href="../resources/pubmed/">infores:pubmed</a></td>
<td>PubMed</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/pubmed-central/">infores:pubmed-central</a></td>
<td>PubMed Central</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/quickgo/">infores:quickgo</a></td>
<td>QuickGO</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/reactome/">infores:reactome</a></td>
<td>Reactome</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/renci-bionames/">infores:renci-bionames</a></td>
<td>RENCI BioNames</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/repodb/">infores:repodb</a></td>
<td>Drug Repositioning Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/rgd/">infores:rgd</a></td>
<td>RGD API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The RGD API</td>
</tr>
<tr>
<td><a href="../resources/ro/">infores:ro</a></td>
<td>Relations Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/robokop/">infores:robokop</a></td>
<td>ROBOKOP</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>An API for answering biomedical questions</td>
</tr>
<tr>
<td><a href="../resources/robokop-kg/">infores:robokop-kg</a></td>
<td>ROBOKOP KG</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/robokop-kp/">infores:robokop-kp</a></td>
<td>ROBOKOP (Reasoning Over Biomedical Objects linked in Knowledge-Oriented Pathway)</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/robokp-ranker/">infores:robokp-ranker</a></td>
<td>ROBOKOP Ranker</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>An API for answering biomedical questions</td>
</tr>
<tr>
<td><a href="../resources/rtx-kg2/">infores:rtx-kg2</a></td>
<td>RTX KG2</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>TRAPI 1.1 endpoint for the NCATS Biomedical Transl...</td>
</tr>
<tr>
<td><a href="../resources/rxnorm/">infores:rxnorm</a></td>
<td>RxNorm</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>normalized names for clinical drugs and links its ...</td>
</tr>
<tr>
<td><a href="../resources/sabio-rk/">infores:sabio-rk</a></td>
<td>SABIO-RK Biochemical Reaction Kinetics Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/scibite/">infores:scibite</a></td>
<td>Semantic Analytics &amp; Clean Data Intelligence Experts</td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/semmeddb/">infores:semmeddb</a></td>
<td>Semantic Medline Database</td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/sgd/">infores:sgd</a></td>
<td>Saccharomyces Genome Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/shepherd-aragorn/">infores:shepherd-aragorn</a></td>
<td>Shepherd Aragorn ARA</td>
<td>released</td>
<td>prediction</td>
<td>Shepherd Aragorn: The Aragorn ARA implementation w...</td>
</tr>
<tr>
<td><a href="../resources/retriever/">infores:retriever</a></td>
<td>Retriever</td>
<td>released</td>
<td>mixed</td>
<td>Retriever: The TRAPI access layer to Tier 0/1/2 KG...</td>
</tr>
<tr>
<td><a href="../resources/sider/">infores:sider</a></td>
<td>SIDER</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/signor/">infores:signor</a></td>
<td>SIGNOR 3.0</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/smart-api/">infores:smart-api</a></td>
<td>SmartAPI API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>This is the API provided from SmartAPI, https://sm...</td>
</tr>
<tr>
<td><a href="../resources/smartapi/">infores:smartapi</a></td>
<td>SmartAPI API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>This is the API provided from SmartAPI, https://sm...</td>
</tr>
<tr>
<td><a href="../resources/smpdb/">infores:smpdb</a></td>
<td>Small Molecule Pathway Database (SMPDB)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/snomedct/">infores:snomedct</a></td>
<td>Systemized Nomenclature of Medicine</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/snpeff/">infores:snpeff</a></td>
<td>SnpEff</td>
<td>released</td>
<td>prediction</td>
<td>Genetic variant annotation and functional effect p...</td>
</tr>
<tr>
<td><a href="../resources/so/">infores:so</a></td>
<td>Sequence Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/spoke/">infores:spoke</a></td>
<td>SPOKE KP</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>SPOKE KP - an NIH NCATS Knowledge Provider to expo...</td>
</tr>
<tr>
<td><a href="../resources/sri-edge-normalization-service/">infores:sri-edge-normalization-service</a></td>
<td>Edge Normalization Service</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>SRI service to validate and normalize edges to Tra...</td>
</tr>
<tr>
<td><a href="../resources/sri-name-resolver/">infores:sri-name-resolver</a></td>
<td>Name Resolver</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Service to resolve Translator concept names.</td>
</tr>
<tr>
<td><a href="../resources/sri-node-normalizer/">infores:sri-node-normalizer</a></td>
<td>Node Normalization</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Node normalization takes a CURIE, and returns reso...</td>
</tr>
<tr>
<td><a href="../resources/sri-ontology/">infores:sri-ontology</a></td>
<td>Ontology-KP API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>TRAPI interface to integrated ontology knowledgeba...</td>
</tr>
<tr>
<td><a href="../resources/sri-reference-kg/">infores:sri-reference-kg</a></td>
<td>SRI Reference Knowledge Graph API</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>The SRI reference KG aims to aggregate knowledge s...</td>
</tr>
<tr>
<td><a href="../resources/sri-semmeddb/">infores:sri-semmeddb</a></td>
<td>SRI SemMedDb KGX File Set</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>The SRI Semantic Medline Database is a Biolink Mod...</td>
</tr>
<tr>
<td><a href="../resources/startgeo/">infores:startgeo</a></td>
<td>STAR GEO Search Application</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/stitch/">infores:stitch</a></td>
<td>STITCH</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Database of known and prediction interactions betw...</td>
</tr>
<tr>
<td><a href="../resources/strider/">infores:strider</a></td>
<td>Strider</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>Translator Autonomous Relay Agent</td>
</tr>
<tr>
<td><a href="../resources/string/">infores:string</a></td>
<td>Search Tool for the Retrieval of Interacting Genes/Proteins</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/symp/">infores:symp</a></td>
<td>Symptom Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/tcga/">infores:tcga</a></td>
<td>The Cancer Genome Atlas</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/tcrd/">infores:tcrd</a></td>
<td>Target Central Resource Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/tiga/">infores:tiga</a></td>
<td>Target Illumination GWAS Analytics</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Aggregating and assessing experimental evidence fo...</td>
</tr>
<tr>
<td><a href="../resources/textmining-kp/">infores:textmining-kp</a></td>
<td>Text Mining KP</td>
<td>deprecated</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/text-mining-provider-cooccurrence/">infores:text-mining-provider-cooccurrence</a></td>
<td>Text Mining COOCCURRENCE API</td>
<td>released</td>
<td>not_provided</td>
<td>API serving a knowledge graph comprised of pairs o...</td>
</tr>
<tr>
<td><a href="../resources/text-mining-provider-targeted/">infores:text-mining-provider-targeted</a></td>
<td>Text Mining Targeted Association API</td>
<td>released</td>
<td>not_provided</td>
<td>API serving explicitly targeted Biolink Associatio...</td>
</tr>
<tr>
<td><a href="../resources/tissues-expression-db/">infores:tissues-expression-db</a></td>
<td>Tissue Expression Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/topmed/">infores:topmed</a></td>
<td>NHLBI Trans-Omics for Precision Medicine</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ttd/">infores:ttd</a></td>
<td>Therapeutic Target Database (TTD)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>TTD is a database providing information about the ...</td>
</tr>
<tr>
<td><a href="../resources/ubergraph/">infores:ubergraph</a></td>
<td>Ubergraph</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A graph representation of Ubergraph, an integratio...</td>
</tr>
<tr>
<td><a href="../resources/uberon/">infores:uberon</a></td>
<td>Uber Anatomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/umls/">infores:umls</a></td>
<td>Unified Medical Language System</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/umls-metathesaurus/">infores:umls-metathesaurus</a></td>
<td>UMLS Metathesaurus (MTH) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/unc-cdwh-ehr-data/">infores:unc-cdwh-ehr-data</a></td>
<td>UNC Carolina Data Warehouse for Health Patient EHR Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/unichem/">infores:unichem</a></td>
<td>UniChem</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/uniprot/">infores:uniprot</a></td>
<td>UniProt Knowledgebase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/uniref/">infores:uniref</a></td>
<td>UniRef</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/unsecret-agent/">infores:unsecret-agent</a></td>
<td>Unsecret Agent OpenAPI for NCATS Biomedical Translator Reasoners</td>
<td>released</td>
<td>prediction</td>
<td>OpenAPI for NCATS Biomedical Translator Reasoners</td>
</tr>
<tr>
<td><a href="../resources/us-census-acs-data/">infores:us-census-acs-data</a></td>
<td>United States Census Bureau American Community Survey Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/us-census-tiger-roadway-exposures-data/">infores:us-census-tiger-roadway-exposures-data</a></td>
<td>United States Census Bureau TIGER/line Roadway Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/us-dot-roadway-exposures-data/">infores:us-dot-roadway-exposures-data</a></td>
<td>United States Department of Transportation Roadway Exposures Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/us-epa-airborne-pollutant-exposures-data/">infores:us-epa-airborne-pollutant-exposures-data</a></td>
<td>United States Environmental Protection Agency Airborne Pollutant Exposures Data</td>
<td>deprecated</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/upheno/">infores:upheno</a></td>
<td>Unified phenotype ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The uPheno ontology integrates multiple phenotype ...</td>
</tr>
<tr>
<td><a href="../resources/vandf-umls/">infores:vandf-umls</a></td>
<td>Veterans Association National Drug File (VANDF) (from UMLS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/wikidata/">infores:wikidata</a></td>
<td>Wikidata</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/wikidata-garbanzo/">infores:wikidata-garbanzo</a></td>
<td>Wikidata Garbanzo</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/wikipathways/">infores:wikipathways</a></td>
<td>WikiPathways</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>WikiPathways is an open science platform for biolo...</td>
</tr>
<tr>
<td><a href="../resources/xpo/">infores:xpo</a></td>
<td>Xenopus Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>XPO represents anatomical, cellular, and gene func...</td>
</tr>
<tr>
<td><a href="../resources/workflow-runner/">infores:workflow-runner</a></td>
<td>Translator Workflow Runner</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Translator general purpose workflow runner</td>
</tr>
<tr>
<td><a href="../resources/wormbase/">infores:wormbase</a></td>
<td>WormBase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/zfa/">infores:zfa</a></td>
<td>Zebrafish anatomy and development ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A structured controlled vocabulary of the anatomy ...</td>
</tr>
<tr>
<td><a href="../resources/zfs/">infores:zfs</a></td>
<td>Zebrafish Developmental Stages Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An ontology of developmental stages of the Zebrafi...</td>
</tr>
<tr>
<td><a href="../resources/zfin/">infores:zfin</a></td>
<td>Zebrafish Information Network</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ddinter/">infores:ddinter</a></td>
<td>DDinter</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>DDInter is an open-access database specific to dru...</td>
</tr>
<tr>
<td><a href="../resources/ki-database/">infores:ki-database</a></td>
<td>Ki database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Ki database serves as a data warehouse for pub...</td>
</tr>
<tr>
<td><a href="../resources/community-sar/">infores:community-sar</a></td>
<td>Community Structure-Activity Resource</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>CSAR disseminated experimental datasets of crystal...</td>
</tr>
<tr>
<td><a href="../resources/drug-design/">infores:drug-design</a></td>
<td>Drug Design Data Resource</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Drug Design Data Resource (D3R) aims to advanc...</td>
</tr>
<tr>
<td><a href="../resources/aeolus/">infores:aeolus</a></td>
<td>Adverse Event Open Learning through Universal Standardization (AEOLUS)</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A knowledge_assertion and standardized version of ...</td>
</tr>
<tr>
<td><a href="../resources/regl/">infores:regl</a></td>
<td>Richards Effector Gene List</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>An effector index to predict target genes at GWAS ...</td>
</tr>
<tr>
<td><a href="../resources/huri/">infores:huri</a></td>
<td>The Human Reference Protein Interactome Mapping Project</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A human ‘all-by-all’ reference interactome map of ...</td>
</tr>
<tr>
<td><a href="../resources/zp/">infores:zp</a></td>
<td>Zebrafish Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Zebrafish Phenotype Ontology formally defines ...</td>
</tr>
<tr>
<td><a href="../resources/flybase/">infores:flybase</a></td>
<td>FlyBase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>A Database of Drosophila Genes &amp; Genomes</td>
</tr>
<tr>
<td><a href="../resources/fypo/">infores:fypo</a></td>
<td>Fission Yeast Phenotype Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>FYPO is a formal ontology of phenotypes observed i...</td>
</tr>
<tr>
<td><a href="../resources/xenbase/">infores:xenbase</a></td>
<td>Xenbase</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Xenbase's mission is to provide the international ...</td>
</tr>
<tr>
<td><a href="../resources/xao/">infores:xao</a></td>
<td>Xenopus Anatomy Ontology</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Represents the anatomy and development of the Afri...</td>
</tr>
<tr>
<td><a href="../resources/biothings-rare-source/">infores:biothings-rare-source</a></td>
<td>BioThings RARe-SOURCE API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [NCATS RARe...</td>
</tr>
<tr>
<td><a href="../resources/rare-source/">infores:rare-source</a></td>
<td>NCATS RARe-SOURCE</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Integrated Bioinformatics Resource for Rare Diseas...</td>
</tr>
<tr>
<td><a href="../resources/sri-answer-appraiser/">infores:sri-answer-appraiser</a></td>
<td>SRI Answer Appraiser</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/biothings-suppkg/">infores:biothings-suppkg</a></td>
<td>BioThings SuppKG API</td>
<td>released</td>
<td>not_provided</td>
<td>Documentation of the BioThings API for [SuppKG](ht...</td>
</tr>
<tr>
<td><a href="../resources/suppkg/">infores:suppkg</a></td>
<td>SuppKG</td>
<td>released</td>
<td>not_provided</td>
<td>SuppKG contains relations between dietary suppleme...</td>
</tr>
<tr>
<td><a href="../resources/biothings-ddinter/">infores:biothings-ddinter</a></td>
<td>BioThings DDInter API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [DDInter](h...</td>
</tr>
<tr>
<td><a href="../resources/biothings-gtrx/">infores:biothings-gtrx</a></td>
<td>BioThings GTRx API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [Genome-to-...</td>
</tr>
<tr>
<td><a href="../resources/biothings-ttd/">infores:biothings-ttd</a></td>
<td>Biothings Therapeutic Target Database API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [Therapeuti...</td>
</tr>
<tr>
<td><a href="../resources/rampdb/">infores:rampdb</a></td>
<td>RaMPDB</td>
<td>released</td>
<td>mixed</td>
<td>RaMP-DB (Relational database of Metabolomic Pathwa...</td>
</tr>
<tr>
<td><a href="../resources/biothings-agr/">infores:biothings-agr</a></td>
<td>BioThings AGR API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [AGR](https...</td>
</tr>
<tr>
<td><a href="../resources/innatedb/">infores:innatedb</a></td>
<td>InnateDB</td>
<td>released</td>
<td>mixed</td>
<td>InnateDB is a publicly available database of the g...</td>
</tr>
<tr>
<td><a href="../resources/biothings-innatedb/">infores:biothings-innatedb</a></td>
<td>BioThings InnateDB API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the BioThings API for [InnateDB](...</td>
</tr>
<tr>
<td><a href="../resources/iptmnet/">infores:iptmnet</a></td>
<td>iPTMnet</td>
<td>released</td>
<td>mixed</td>
<td>iPTMnet is a bioinformatics resource for integrate...</td>
</tr>
<tr>
<td><a href="../resources/fda-orphan-drug-db/">infores:fda-orphan-drug-db</a></td>
<td>FDA Orphan Drug Project designation database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Resource for searching FDA Orphan Drug Designation...</td>
</tr>
<tr>
<td><a href="../resources/complex-portal/">infores:complex-portal</a></td>
<td>Complex Portal</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Complex Portal is a manually curated, encyclop...</td>
</tr>
<tr>
<td><a href="../resources/biothings-repodb/">infores:biothings-repodb</a></td>
<td>BioThings repoDB API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [repoDB](ht...</td>
</tr>
<tr>
<td><a href="../resources/biothings-hmdb/">infores:biothings-hmdb</a></td>
<td>BioThings HMDB API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [HMDB](http...</td>
</tr>
<tr>
<td><a href="../resources/fooddata-central/">infores:fooddata-central</a></td>
<td>FoodData Central</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>FoodData Central is an integrated data system that...</td>
</tr>
<tr>
<td><a href="../resources/biothings-fooddata-central/">infores:biothings-fooddata-central</a></td>
<td>BioThings FoodData Central API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for  [FoodData ...</td>
</tr>
<tr>
<td><a href="../resources/biothings-foodb/">infores:biothings-foodb</a></td>
<td>BioThings FooDB API</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Documentation of the BioThings API for [FooDB](htt...</td>
</tr>
<tr>
<td><a href="../resources/tissues/">infores:tissues</a></td>
<td>TISSUES</td>
<td>released</td>
<td>mixed</td>
<td>TISSUES is a weekly updated web resource that inte...</td>
</tr>
<tr>
<td><a href="../resources/biothings-tissues/">infores:biothings-tissues</a></td>
<td>BioThings TISSUES API</td>
<td>released</td>
<td>mixed</td>
<td>Documentation of the BioThings API for [TISSUES](h...</td>
</tr>
<tr>
<td><a href="../resources/mygeneset-info/">infores:mygeneset-info</a></td>
<td>MyGeneset.info API</td>
<td>released</td>
<td>other</td>
<td>MyGeneset allows scientists to conveniently collec...</td>
</tr>
<tr>
<td><a href="../resources/gelinea/">infores:gelinea</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-binding-db/">infores:automat-binding-db</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-reactome/">infores:automat-reactome</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/automat-robokop/">infores:automat-robokop</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/ars/">infores:ars</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/chv-umls/">infores:chv-umls</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/annotator/">infores:annotator</a></td>
<td>Annotator Service</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>Service to provide detailed annotations for any gi...</td>
</tr>
<tr>
<td><a href="../resources/dsstoxdb/">infores:dsstoxdb</a></td>
<td>Distributed Structure-Searchable Toxicity (DSSTox) Database</td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/omop-ohdsi/">infores:omop-ohdsi</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/phenopacket-store/">infores:phenopacket-store</a></td>
<td>PhenoPacket Store</td>
<td>released</td>
<td>observation</td>
<td>A repository for storage and retrieval of phenopac...</td>
</tr>
<tr>
<td><a href="../resources/biothings-mabs/">infores:biothings-mabs</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/sepid_mab/">infores:sepid_mab</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/amyco/">infores:amyco</a></td>
<td>The Amyloidoses Collection (AmyCo) Database</td>
<td>released</td>
<td>knowledge_assertion</td>
<td>The Amyloidoses Collection (AmyCo) database contai...</td>
</tr>
<tr>
<td><a href="../resources/catrax-pharmacogenomics/">infores:catrax-pharmacogenomics</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/unii/">infores:unii</a></td>
<td>Unique Ingredient Identifier</td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
<tr>
<td><a href="../resources/fooddb/">infores:fooddb</a></td>
<td></td>
<td>released</td>
<td>not_provided</td>
<td>N/A</td>
</tr>
</tbody>
</table>









  




                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.6ce7567c.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.525ec568.min.js"></script>
      
    
  </body>
</html>